### PR TITLE
XSH-837: Allow returning local ids of header labels

### PR DESCRIPTION
**Pull request by torchbear**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: gooddata-pandas/gooddata_pandas/result_convertor.py [hunk 1]
```diff
@@ -197,19 +197,27 @@ def _read_complete_execution_result(
...
     :param label_overrides: label overrides
+    :param use_local_ids_in_headers: Use local identifiers of header attributes and metrics. Optional.
+
     :return: Mapper function
     """
     if label_overrides is None:
         label_overrides = {}
 
+    if use_local_ids_in_headers is None:
+        use_local_ids_in_headers = False
```
Comment (hkad98) [inline gooddata-pandas/gooddata_pandas/result_convertor.py:219]: Why do not have `use_local_ids_in_headers` set to False? Then you will not need this code.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: gooddata-pandas/gooddata_pandas/result_convertor.py (hunk 1 above)
Comment (torchbear) [reply gooddata-pandas/gooddata_pandas/result_convertor.py]: fixed

File: gooddata-pandas/gooddata_pandas/result_convertor.py [hunk 2]
```diff
@@ -197,14 +197,19 @@ def _read_complete_execution_result(
 
 
 def _create_header_mapper(
-    response: BareExecutionResponse, dim: int, label_overrides: Optional[LabelOverrides] = None
+    response: BareExecutionResponse,
+    dim: int,
+    label_overrides: Optional[LabelOverrides] = None,
+    use_local_ids_in_headers: Optional[bool] = False,
```
Comment (hkad98) [inline gooddata-pandas/gooddata_pandas/result_convertor.py:203]: One last thing, you can remove Optional. Optional says that the value can be None and I think that we do not want that in this case.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by torchbear** [comment]
ok to test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Review by hkad98 — APPROVED** [review]

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-python-sdk-gate-PR158

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-sonarqube-scan-zuul-gate/144/ : SUCCESS in 55s
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-fossa-scanning-zuul-el7/144/ : FAILURE in 14s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-merge-compliance-check/146/ : SUCCESS in 24s

**Pull request merged**